Common/Src/zp_assert.c

```diff
@@ -0,0 +1,17 @@
+#include "zp_assert.h"
+
+static sAssertInfo g_assert_info;
+
+void store_assert(const uint64_t *pc, const uint64_t *lr) {
+    g_assert_info.pc = pc;
+    g_assert_info.lr = lr;
+}
+
+void zp_assert(bool expr) {
+    if (!expr) {
+//#warning This should call the los hardfault handler, or something more appropriate
```

Comment:
If we assert, we should probably also set all motors to 0 throttle. Can you think of any other safety related functions that should be performed in the event if an assert?